src/compiler/scala/tools/nsc/settings/StandardScalaSettings.scala

```diff
@@ -90,7 +93,6 @@ trait StandardScalaSettings { _: MutableSettings =>
       // .withAbbreviation("--Xtarget")
       // .withAbbreviation("-Xtarget")
       .withAbbreviation("-Xunchecked-java-output-version")
-      .withDeprecationMessage("Use -release instead to compile against the correct platform API.")
```

Comment:
conversely, you must specify both target and system? IIRC the idea was not to specify target and assume your client is safe. If you go "fully manual", with both target and system, then you're on your own.

Reply:
Do we have a way of reporting conditional deprecations? We can't make it an error. I tried calling `reporter.withDeprecationMessage` in the post-set hook but it doesn't report anything.

Reply:
D'Oh, `reporter` is completely the wrong thing to look at here. That's just another setting. But I don't see anything else that would be useful. The unconditional deprecations are wrapped in the Settings and enforced in Global. I guess we'd need to add some abstractions for conditional deprecations.

Reply:
https://github.com/scala/scala/blob/v2.13.14/src/compiler/scala/tools/nsc/settings/ScalaSettings.scala#L669-L678 should work for that

Reply:
also `-javabootclasspath` is exclusive of jrt at https://github.com/scala/scala/blob/2.13.x/src/compiler/scala/tools/util/PathResolver.scala#L260 but probably anyone futzing with it would know that.